### PR TITLE
added filename in dependabot

### PR DESCRIPTION
**Pull request by hellspawn679**
## Which problem is this PR solving?
- https://github.com/jaegertracing/jaeger/issues/5552

## Description of the changes
- added` includeds `statement in dependabot.yml  to include docker compose files name 

## How was this change tested?
- 

## Checklist
- [ ] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [ ] I have signed all commits
- [ ] I have added unit tests for the new functionality
- [ ] I have run lint and test steps successfully
  - for `jaeger`: `make lint test`
  - for `jaeger-ui`: `yarn lint` and `yarn test`

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -122,24 +122,36 @@ updates:
     allow:
       - dependency-name: "docker.elastic.co/elasticsearch/elasticsearch"
         update-types: ["version-update:semver-minor"]
+    includeds:
```
Comment (yurishkuro) [inline .github/dependabot.yml:125]: a) you have a typo, b) where is the keyword "includes" documented?

Comment (yurishkuro) [reply .github/dependabot.yml:125]: c) would we still need `directory` if this works?

**Review by yurishkuro — COMMENTED** [review]
does dependabot provide a way of testing config file changes?

**Comment by hellspawn679** [comment]
> does dependabot provide a way of testing config file changes?

**Pull request closed without merging**